fire/core.py

```diff
@@ -132,7 +131,7 @@ def Fire(component=None, command=None, name=None):
     result = component_trace.GetResult()
     print(
         helputils.HelpString(result, component_trace, component_trace.verbose))
-    raise FireExit(2, trace)
+    raise FireExit(2, component_trace)
   elif component_trace.show_trace and component_trace.show_help:
     print('Fire trace:\n{trace}\n'.format(trace=component_trace))
     result = component_trace.GetResult()
```

Comment:
Think we should `raise FireExit(0, component_trace)` at lines 140 and 143 as well (the successful trace and successful help+trace cases)?